app/utils/tree.js

```diff
@@ -1 +0,0 @@
-export { buildTree, buildFlatList, getDescendents, getAncestors, getParent } from 'x-tree/utils/tree';
```

Comment:
These are useful, we should leave them so other applications or addons can import them and build the model for this addon

Reply:
Other applications can import them without a reexport like this:

`import { buildTree } from 'ember-simple-tree/utils/tree';`

What won't work anymore is:

`import { buildTree } from '<application-name>/utils/tree';`

or

`this.owner.lookup('util:tree').buildTree(...);`

I think this is a good thing.

Reply:
I agree that is better the first example